content/guides/models/app/settings-page/team-settings.md

```diff
@@ -43,10 +43,11 @@ For more information about how to set up and manage alerts, see [Send alerts wit
 
 ## Privacy
 
-Navigate to the **Privacy** section to change privacy settings. Only organization admins can modify privacy setting, including:
+Navigate to the **Privacy** section to change privacy settings. Only organization admins can modify privacy setting.
 
-* Force projects in the team to be private.
-* Enable code saving by default.
+- Optionally disable the ability to make future projects public or to share reports publicly.
```

Comment:
```suggestion
- Optionally enable the ability to make future projects public or to share reports publicly.
```

Reply:
I think you're reading it backward, this is about turning that off. I'll edit your edit.